### ISSUE TITLE
Universal viewer iframe must have title attribute.

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#1227]
https://catalog.princeton.edu/catalog/4782614

element location: "#easyXDM_default6273_provider"       

<iframe name="easyXDM_default6273_provider" id="easyXDM_default6273_provider" ....>

**Comment by axamei** [comment]
the iframe id also needs to have different values. currently, the iframe produces `id="iframe"`. this is problematic when a record has more than 1 viewer, as id attribute values must be unique.

**Comment by axamei** [comment]
Other wcag 2.0 violations related to the universal viewer (using this record http://localhost:3000/catalog/3943643z). Location of elements with these violations listed below.

## Critial violations:
- Form elements must have labels (critical violation)
  - #view > iframe:nth-child(1) .image-selectionbox
  - iframe:nth-child(2) .image-selectionbox
  - #iframe_1 .image-selectionbox
- Zooming and scaling must not be disabled (critial violation)
  - #view > iframe:nth-child(1) meta
  - iframe:nth-child(2) meta
  - #iframe_1 meta

## Serious violations:
- `<html>` element must have a lang attribute
  - #view > iframe:nth-child(1) html
  - iframe:nth-child(2) html
  - #iframe_1 html
- Links must have discernible text
  - #view > iframe:nth-child(1) #top
  - iframe:nth-child(2) #top
  - #iframe_1 #top

**Comment by sandbergja** [comment]
> the iframe id also needs to have different values. currently, the iframe produces `id="iframe"`. this is problematic when a record has more than 1 viewer, as id attribute values must be unique.

This was resolved by d9a16d14c5aa8456d32f3236226b1941ec1f65fb

The other issues raised in [this comment](https://github.com/pulibrary/orangelight/issues/1227#issuecomment-413947951) are still valid, so I've split them into a separate ticket: https://github.com/pulibrary/orangelight/issues/3145
